src/components/ItemListButton.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+function ItemListButton() {
+  const history = useHistory();
+
+  const handleRedirectToList = () => {
+    history.push('/list');
+  };
+  return (
+    <div className="hidden md:block">
+      <div className="fixed bottom-12 left-1/4">
+        <button
+          type="submit"
+          onClick={handleRedirectToList}
+          className="btn-bg flex items-center border py-3 px-3 hover:bg-green-300 justify-center rounded-md bg-green-600 text-white w-36"
+        >
+          <span>
```

Comment:
Why not just using a` double arrow` icon? 😆

Reply:
fixed!👍🏻